safeeyes/ui/break_screen.py

```diff
@@ -243,8 +252,11 @@ def __update_count_down(self, count):
 
     def __lock_keyboard(self):
         """
-        Lock the keyboard to prevent the user from using keyboard shortcuts
+        Lock the keyboard to prevent the user from using keyboard shortcuts (X11 only)
         """
+        if self.is_wayland:
+            return
+
```

Comment:
Is this extra check necessary? From what I can see, this method will not be called when on wayland (same with `__release_keyboard`).